models/train.js

```diff
@@ -0,0 +1,89 @@
+// Things to ask:
+
+// ## Train
+
+// "determine which train is arriving next at a particular station"
+//    Does this belong on the train model
+//    Ask about order of trains
+//      Keep track of arival/departure times??
+
+
+// ## Station
+
+// difference between these:
+//    get the passengers waiting for a train at a particular station.
+//    get the passengers who have tickets at a particular station.
+//      can you be waiting for a train without a ticket???
+
+// determine which is the next train arriving at a particular station.
+//    is this differenct from the one on the train model?
+
+
+const stations = ['Downtown', 'Elm Street', 'Forest Gardens', 'Annex', '10th Ave', 'Waterfront', 'Colosseum', 'Central Station', 'Parkside', 'Grand Boulevard', 'Monument Valley', 'Museum Isle']
+
+export default class Train {
+  constructor(options){
+    this.number = options.number || 0
+    this.capacity = options.capacity || 200
+    this.passengers = options.passengers || [{id: 00, destination: 3}]
+    this.currentStationIndex = options.currentStationIndex || 0
+
+    db.updateTrain({
```

Comment:
This shouldn't happen in the constructor of a class. Why are you updating the train when you create a new instance of Train?